pandas/tests/series/test_period.py

```diff
@@ -113,3 +113,13 @@ def test_to_period(self, input_vals):
         expected = Series([input_vals], dtype="Period[D]")
         result = Series([input_vals], dtype="datetime64[ns]").dt.to_period("D")
         tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize("comparison_data", ["a", False, 1, 1.0, None])
```

Comment:
can you call this "other" or "scalar"

test belongs in tests.arithmetic.test_period